syft/serde/protobuf/proto.py

```diff
@@ -56,6 +57,7 @@
     torch.Size: SizePB,
     # Syft types
     AdditiveSharingTensor: AdditiveSharingTensorPB,
+    ObjectId: IdPB,  # TODO harmonize naming between syft and proto
```

Comment:
One way to do this without breaking everything would be to create an `ObjectId`/`PlaceholderId` schema on the Protobuf side that contains an `Id` field.

Reply:
How bad would it be to rename protobuf's `Id` to `ObjectId`/`PlaceholderId`?

Reply:
Kinda bad? It's not really intended for either of those specific purposes.

Reply:
Even if ObjectId was used everywhere?

Reply:
If we were going to do that, we'd probably want to rename the Python class to `Id`, because I'm not sure everything that uses an `Id` is actually using it as the id for an object.

Reply:
(Still seems like we might want a way to immediately tell what type an id goes with from the class though.)

Reply:
I don't really get why we would need to know which type the goes to 🤔

Reply:
Two reasons I can think of:
- It's handy for serialization/deserialization.
- It prevents people from misusing the ids from one class to load objects of another class (to prevent weird collisions of both accidental and intentional natures.)

Reply:
- Do you think serialization/deserialization could differ between different id classes?
- Hum, I agree on this point

Reply:
Depends on whether they're represented by separate classes, I suppose. If they are, then seems like the serialization could be different.